pkgs/development/python-modules/atpublic/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+# python3 only
+buildPythonPackage rec {
+  pname = "atpublic";
+  version = "1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0i3sbxkdlbb4560rrlmwwd5y4ps7k73lp4d8wnmd7ag9k426gjkx";
+  };
+
+  # TypeError: calling <function public at 0x7ffff26dbb70> returned None, not a test
+  doCheck = false;
```

Comment:
That should be fixed.

Reply:
Don't know if this is how it should be done but running `pytest --pyargs public` in `checkPhase` fixes it.